PixelDefinitions/pixels/subscription_pixels.json5

```diff
@@ -13,5 +13,24 @@
                 "enum": ["true", "false"]
             }
         ]
+    },
+    "m_privacy-pro_app_subscription_active": {
+        "description": "Fired once daily on app start for an active subscription",
+        "owners": ["lmac012"],
+        "triggers": ["other"],
+        "suffixes": ["form_factor"],
+        "parameters": [
+            "appVersion",
+            {
+                "key": "os_version",
+                "description": "OS version of the device",
+                "type": "integer"
+            },
+            {
+                "key": "petal",
+                "description": "Signals the pixel processing pipeline",
+                "type": "boolean"
+            }
```

Comment:
I'm not sure about the best practices here, but shouldn't we add `"enum": ["true"]` to indicate that this is the only possible value for the `petal` param? cc @nshuba 

Regardless, I think it would make sense to add this parameter to `params_dictionary.json`, since it's not specific to this pixel and will likely get reused on other pixels as well.